wallet/src/actors/worker/methods.rs

```diff
@@ -1013,7 +1015,13 @@ impl Worker {
         wallet: types::SessionWallet,
         sink: types::DynamicSink,
     ) -> Result<()> {
-        log::debug!("The node has changed its status into {:?}", status);
+        log::info!("The node has changed its status into {:?}", status);
+        if status == StateMachine::Synced && !wallet.is_syncing()? {
```

Comment:
Note that if the node sends a notification when the state changes from `Synced` to `Synced`, this will trigger an unneeded synchronization. We can prevent that from happening in the node, or we can detect if the wallet is more than N blocks behind the node, and only sync if we see that it is behind.

Reply:
I think this does not happen, the StateMachine is updated to Synced only when consensus_beacon is equal to our_beacon and the StateMachine is AlmostSynced, so it can not happen that "changes" from Synced to Synced